tests/test_daemon.cpp

```diff
@@ -266,6 +266,10 @@ struct Daemon : public Test
 
         ON_CALL(*mock_factory_ptr, get_backend_version_string()).WillByDefault(Return("mock-1234"));
 
+        ON_CALL(*mock_factory, list_networks())
```

Comment:
Tiny default, could you use the same as other calls in here? Not that dereferencing a `unique_ptr` has any significant impact, but just for consistency, so that the reader does not go wonder what is what.

Reply:
Oops, I didn't note this. Maybe it was an auto-complete. Fixed.